tests/unit/metadata/test_multi_table.py

```diff
@@ -1718,7 +1721,7 @@ def test_load_from_json_path_does_not_exist(self, mock_path):
             - String representing a filepath.
 
         Side Effects:
-            - A ``ValueError`` is raised pointing that the ``file`` does not exist.
+            - An ``InvalidMetadataError`` is raised pointing that the ``file`` does not exist.
```

Comment:
This test raises a ValueError, so you shouldn't update the docstring.

Reply:
According to our new testing docstring guidelines I think you can actually just delete all the docstrings heere after line 1715